### ISSUE TITLE
accessibility checks

### ISSUE THREAD

**Issue by chrissycooper** [chrissycooper/overlook#22]
Accessibility audits should be at 100% for the dashboard.
A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
